src/components/vr_cooperation/src/vr_module.cc

```diff
@@ -240,17 +240,11 @@ void VRModule::SendMessageToHMI(const vr_hmi_api::ServiceMessage& msg) {
   if (!proxy_.Send(msg)) {
     LOG4CXX_ERROR(logger_, "Couldn't send GPB message");
   }
-  if(vr_hmi_api::RESPONSE == msg.rpc_type()) {
```

Comment:
when response is timeout request already deleted when OnTimeoutTriggered() of request_controller